src/components/EditFreakForm/EditFreakForm.jsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import { Form, Row, Col } from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import { FreakModelDefault, FreakModelKeys } from '../../models/freaks';
+
+function EditFreakForm({ freak, onChange, onSubmit }) {
+  function triggerChange(name) {
+    return ({ target: { value } }) => {
+      const newFreak = { ...freak, [name]: value };
+      onChange(newFreak);
+    };
+  }
+
+  return (
+    <Form id="add-freak-form" className=" mx-2 p-3" onSubmit={ onSubmit }>
+      <Form.Group className="mb-3">
+        <Form.Label>Your name*</Form.Label>
+        <Row>
+          <Col>
+            <Form.Control
+              required
+              data-testid="first-name-input"
+              placeholder="First name"
+              value={ freak.firstName }
+              onChange={ triggerChange(FreakModelKeys.firstName) }
+            />
+          </Col>
+          <Col>
+            <Form.Control
+              required
+              data-testid="last-name-input"
+              placeholder="Last name"
+              value={ freak.lastName }
+              onChange={ triggerChange(FreakModelKeys.lastName) }
+            />
+          </Col>
+        </Row>
+      </Form.Group>
+
+      <Form.Group className="mb-3" controlId="formBasicEmail">
```

Comment:
What is the `controlId="formBasicEmail"` used for?

Reply:
Is from bootstrap, it seems I don't need him, I deleted this.